lib/plugins/aws/package/compile/events/cloudFront/index.js

```diff
@@ -97,6 +97,7 @@ class AwsCompileCloudFrontEvents {
 
     const origins = [];
     const behaviors = [];
+    let isDefaultOrigin;
```

Comment:
This one should be kept as `defaultOrigin`, as it hosts origin string and not a boolean